src/main/java/io/stargate/sgv2/jsonapi/api/model/command/CommandContext.java

```diff
@@ -23,7 +24,8 @@ public record CommandContext<T extends SchemaObject>(
     EmbeddingProvider embeddingProvider,
     String commandName,
     JsonProcessingMetricsReporter jsonProcessingMetricsReporter,
-    ApiFeatures apiFeatures) {
+    ApiFeatures apiFeatures,
```

Comment:
+1 for sneaking this in here, will make it easier to remove all the injection of it

Reply:
That's the idea yes. I really dislike the part of it being next to impossible to create test version of `OperationsConfig` (it being interface with magically generated implementation that merges in defaults from annotations).
But at least we can isolate magic, and eventually replace it.